.github/workflows/admin.yml

```diff
@@ -37,7 +32,10 @@ jobs:
           firebase setup:emulators:ui
 
       - name: Install Dependencies
-        run: npm --workspace=@socialincome/admin install
+        run: |
+          npm --workspace @socialincome/functions install \
```

Comment:
Nitpik: At this point I wonder if it wouldn't be better to just run 'npm ci' (installs all workspaces) at all times and cache the .npm folder correctly. I think that is probably faster than 3 separate installs.

Reply:
@andrashee, why did you have to change this at all? Shouldn't installing the admin dependencies be enough?

Reply:
Not required any more when we remove the functions dependencies from the tests 👍